address/list.js

```diff
@@ -0,0 +1,96 @@
+var addr_default = [5,6,7,8];
+
+var loc = window.location.pathname;
+var split_url = loc.split('/');
+var url_host1=window.location.origin+"/"+split_url[1];//+"/"+split_url[2];
+
+$(document).ready(function () {
+    $('[data-toggle="tooltip"]').tooltip();
+    $('#create').on('click', function () {
+        $('.show-create').html('Create Service Request !');
+        $('.modal-create').modal('show');
+    });
+
+    $('#add_addr').on('click', function(){
+        $('.show-addr').html('Filter Date');
+        $('.modal-map-addr').modal('show');
+    })
+
+});
+function checklist_tb(tstr, tstp) { //ditambahkan start dan stop date
+    datatable_list = $("#checklist").DataTable({
+      "ajax": apiUrl +'/cu?para=list_thermo&tstr='+tstr+'&tstp='+tstp,
+      "dom": '<"top"<"dt-filters"f>>rFt<"dt-bottom"<"dt-information"li><"dt-pagination"p>>',
+      buttons: [ 'colvis' ],
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/davidgunadi/hello-world/pullRequest?prid=8393720)
```suggestion
      buttons: [ "colvis" ],
```